backend/src/utils/constants.ts

```diff
@@ -58,12 +62,17 @@ export const blankDashboardCR: DashboardConfig = {
       disableKServe: false,
       disableModelMesh: false,
       disableAcceleratorProfiles: false,
+      disableDistributedWorkloads: true,
     },
     notebookController: {
       enabled: true,
+      ...(isRHOAI && {
+        pvcSize: '20Gi',
+        notebookNamespace: 'rhods-notebooks',
+      }),
     },
     groupsConfig: {
-      adminGroups: 'odh-admins',
+      adminGroups: isRHOAI ? 'rhoai-admins' : 'odh-admins',
```

Comment:
so we do not not use `dedicated-admins` ?

Reply:
@zdtsw what do you guys do behind here?
https://github.com/red-hat-data-services/odh-manifests/blob/master/odh-dashboard/odhdashboardconfig/odhdashboardconfig.yaml#L27

Reply:
If you were replacing this with dedicated-admins i can change it. Based on a conversation on slack ai thought it was changed to rhoai-admins.

Reply:
@andrewballantyne We previously defined them here https://github.com/red-hat-data-services/rhods-operator/pull/194/files#diff-3ddcc8f65a628050fbf81b0ede482ebe52b1418631bf8586c509916b8b767dcdL215

Essentially depending on the platform set the values of admins-group and then replace it in yaml file

Reply:
```
		deploy.SelfManagedRhods: "rhods-admins",
		deploy.ManagedRhods:     "dedicated-admins",
```
Ah -- there it is -- two types of deployments

Reply:
Mmm yes, then i think it's better to keep that logic, I'm gonna revert the changes and i can sync with @zdtsw first thing in the morning to get both PRs

Reply:
Ok @zdtsw I've recovered the old `odhDashboardConfig` from rhoai, it should be the same in this route:

`manifests/overlays/odhdashboardconfig`